packages/components/addon/components/hds/code-block/index.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { assert } from '@ember/debug';
+import { htmlSafe } from '@ember/template';
+import { guidFor } from '@ember/object/internals';
+
+import Prism from 'prismjs';
+
+import 'prismjs/plugins/line-numbers/prism-line-numbers';
+import 'prismjs/plugins/line-highlight/prism-line-highlight';
+
+import 'prismjs/components/prism-bash';
+import 'prismjs/components/prism-go';
+import 'prismjs/components/prism-markup-templating';
+import 'prismjs/components/prism-handlebars';
```

Comment:
These two were added to overcome [an issue where two instances of Prism clash in our documentation website](https://github.com/hashicorp/design-system/pull/1691#discussion_r1367359230). Difficult to tell, but `prism-handlebars` depends on `prism-markup-templating` (it took me a while to figure that out), that's why we need both.

Reply:
Updated in [4d2fa38](https://github.com/hashicorp/design-system/pull/1687/commits/4d2fa380b822b0ffa79a43ebfe5df61d078849bb)